sign_oca/models/sign_oca_request.py

```diff
@@ -413,18 +413,21 @@ def _onchange_role_id(self):
     def get_info(self, access_token=False):
         self.ensure_one()
         self._set_action_log("view", access_token=access_token)
+        partner_fields_dict = {"id": self.partner_id.id}
```

Comment:
I don't like this approach as we are getting all the data of the partner....

My suggestion would be to match with the possible field names.  WDYT?

Reply:
A better approach could be to make only visible the fields availables to group base_user? If there were some.
A mere list seems a hard approach, or not?

Reply:
Yes, but it could be actually a portal user :thinking: 

The thing is that it is more limited to attach to the fields that we have defined

Reply:
I added a `with_user` to ensure only fields accessible to the user opening the form are added.
Do you think this is enough?